lib/krane.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/common'
```

Comment:
This is a partial optimization to make loading time faster. If we require `kubernetes-deploy` it takes about 0.8 seconds to print the `krane help`. As is about 0.4 and if we require nothing 0.3. I feel like this strikes a good balance between not feeling to slugish to print help but not have to cherry pick exactly what we want when want it.